packages/main/src/Calendar.ts

```diff
@@ -52,6 +52,12 @@ interface ICalendarPicker {
 	_lastYear?: number,
 }
 
+/**
+ * Interface for components that may be slotted inside a `ui5-calendar`.
+ *
```

Comment:
As agreed in Teams, I suggest:
**Note:** Use with `ui5-date` or `ui5-calendar-date-range` as calendar date selection types. Implementing the interface does not guarantee that any other class can work as a calendar date selection type.